src/config.js

```diff
@@ -16,8 +16,8 @@ export const sidebarWidth = {
 
 export const chartMinWidthScale = 0.25;
 
-// this value is used to determine the amount of nodes in pipeline to trigger chonky warning
-export const chonkyNodeAmount = 1000;
+// this value is used to determine the amount of nodes in pipeline to trigger large warning
```

Comment:
I think this comment should mention 'nodes and edges'  if it's reflecting the threshold function?

Reply:
updated the comment to reflect the suggested changes